vertx-web/src/test/java/io/vertx/ext/web/WebTestBase.java

```diff
@@ -37,13 +37,15 @@
   protected HttpServer server;
   protected HttpClient client;
   protected Router router;
+  protected int DEFAULT_PORT;
```

Comment:
@tsegismont Now the ports (8080 -> HTTP1x ) and (8181 -> HTTP2) using h2c as you recommend for testing